src/extensionConfiguration.ts

```diff
@@ -0,0 +1,16 @@
+import { loadConfigurationForExtension } from '@codingame/monaco-editor-wrapper/dist/features/extensionConfigurations'
+import staticOptions, { LanguageClientId } from './staticOptions'
+
+export async function loadExtensionConfigurations (forLanguageClientIds: LanguageClientId[], nonMutualizableOnly: boolean): Promise<void> {
+  const extensionConfigurationToLoad = new Set<string>()
+  for (const languageClientId of forLanguageClientIds) {
+    const config = staticOptions[languageClientId]
+    if (!config.mutualizable || !nonMutualizableOnly) {
```

Comment:
do you still need to check `config.mutualizable` if you have this new boolean?

Reply:
what do you mean?

Reply:
b2861e2d7d80f6a1b9ed422c2de75c3ec54debce ?